build-images/Dockerfile.sbom-deployer

```diff
@@ -0,0 +1,3 @@
+FROM docker-all.repo.sonatype.com/python:3.12
+
+RUN apt-get update && apt-get install -y jq curl
```

Comment:
Nit: new line missing

Reply:
Resolved in [ee38b2e](https://github.com/sonatype/docker-nexus3/pull/191/commits/ee38b2e875b7b008d5a718f4a54727534e660e56)